Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -1,107 +1,51 @@
-#ifndef Geometry_CommonTopologies_SimplePixelTopology_h
-#define Geometry_CommonTopologies_SimplePixelTopology_h
+#ifndef CUDADataFormats_TrackerGeometry_SimplePixelTopology_h
```

Comment:
Why the change in the include guard? Did you intent to move the file to `CUDADataFormats/TrackerGeometry/interface/SimplePixelTopology.h` ?